README.rst

```diff
@@ -19,12 +19,12 @@ much faster to develop and enjoyable thanks to:
 
 **Peter Harrison**
   Who posts many useful and detailed resources in `micromouseonline.com
-  <http://micromouseonline.com>`__, `GitHub
-  <https://github.com/micromouseonline>`__, and who is always willing to help.
+  <http://micromouseonline.com>`__ and on his `micromouseonline GitHub
+  <https://github.com/micromouseonline>`__ profile and who is always willing to help.
 **Green Ye**
-  Who shares `details about his micromice <http://www.greenye.net/>`__ and
+  Who shares details about his `micromouse <http://www.greenye.net/>`__ and
```

Comment:
I think the correct term is "micromice" (since it is plural).

Also, I prefer the link description as it was before "details about his micromice", since it is more explicit.